src/cr/cube/stripe/cubemeasure.py

```diff
@@ -92,6 +124,15 @@ def means(self):
         return self._means
 
 
+class _CatCubeOverlaps(_BaseCubeOverlaps):
+    """Overlaps cube-measure for a non-MR stripe."""
+
+    @lazyproperty
+    def overlaps(self):
+        """1D np.float64 ndarray of cube overlaps for each stripe row."""
+        return self._means
```

Comment:
i think this is overlaps isn't ? self._means is a copy paste typo i guess

Reply:
ditched the entire class since we don't need it